src/net/bus.c

```diff
@@ -732,10 +778,10 @@ void BUS_get_var_values(Bus* bus, Vec* values, int code) {
     if (bus->vars & BUS_VAR_VMAG) {
       switch (code) {
       case UPPER_LIMITS:
-	VEC_set(values,bus->index_v_mag[t],bus->v_max);
+	VEC_set(values,bus->index_v_mag[t],bus->v_max_reg);
 	break;
       case LOWER_LIMITS:
-	VEC_set(values,bus->index_v_mag[t],bus->v_min);
+	VEC_set(values,bus->index_v_mag[t],bus->v_min_reg);
```

Comment:
As discussed, regulation limits should not be used here (for upper and lower limits)